content/BCBestPractices/Readability/begin-as-an-afterword/index.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "begin as an 'After Word'"
+weight = 230
++++
+
+<_Created by Microsoft, Described by waldo_\>
+
+## Description
+
+When `begin` follows `then`, `else`, `do`, it should be on the same line, preceded by one space character. 
+
+## Bad code
+
+```al
+    if ICPartnerRefType = ICPartnerRefType::"Common Item No." then
+    begin
+    ...
+    end;
+```
+
+## Good code
+
+```al
+    if ICPartnerRefType = ICPartnerRefType::"Common Item No." then begin
+    ...
+    end;
+```
+
```

Comment:
How do we handle multi-line conditions?

```AL
if (ICPartnerRefType = ICPartnerRefType::"Common Item No.") and
   (ICPartnerRefType::"Common Item No." <> '')
then begin
   blah
end;
```

or

```AL
if (ICPartnerRefType = ICPartnerRefType::"Common Item No.") and
   (ICPartnerRefType::"Common Item No." <> '') then begin
     ...
     blah
end;
```

Reply:
I'd say:

```al
if (ICPartnerRefType = ICPartnerRefType::"Common Item No.") and
   (ICPartnerRefType::"Common Item No." <> '') then begin
     ...
     blah
end;
```

Reply:
In Fact - there is a guideline about this - and I'm wrong :D - I'll attach the PR.

Reply:
Here you go: https://github.com/microsoft/alguidelines/pull/75